Intersect.Server/Entities/Entity.cs

```diff
@@ -1708,23 +1708,7 @@ public virtual void TryAttack(Entity target)
                 return;
             }
 
-            //Check for enemy statuses
-            foreach (var status in enemy.CachedStatuses)
-            {
-                //Invulnerability ignore
-                if (status.Type == StatusTypes.Invulnerable)
-                {
-                    PacketSender.SendActionMsg(enemy, Strings.Combat.invulnerable, CustomColors.Combat.Invulnerable);
```

Comment:
You're losing this message, but not sending it anywhere down the line?
The client will never know the entity they're hitting went invulnerable.

Reply:
It should show up as an action message when the Invulnerability takes effect/is casted -- and invulnerability would be a relatively short buff due to its nature. If a player/mob/whatever is getting hit by 100 spells/second we don't want to send the invulnerable action message hundreds of times.